app/ui/base/src/main/res/values/dimensions.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="toolbarTitleMarginVertical">8dp</dimen>
```

Comment:
The Material 3 spec doesn't seem to explicitly [specify](https://m3.material.io/components/top-app-bar/specs) this for the small top app bar. However, the app bar height is given as 64dp and the icon size as 48dp x 48dp. That leaves 16dp. Using an equal size for top and bottom padding we arrive at 8dp.

Reply:
It is vertically centered and title and subtitle are not officially supported in the Material 3 spec. We need to check with design if we still want to support this.